test/built-ins/Temporal/PlainDateTime/from/argument-plaindatetime.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.from
+description: A Temporal PlainDateTime object is an acceptable argument
```

Comment:
There's a similar test, https://github.com/tc39/test262/blob/main/test/built-ins/Temporal/PlainDate/from/argument-plaindate.js that has a description "A PlainDate object is copied, not returned directly" which I think tells a bit more about what we are testing here. I'm basing the corresponding test for Instant on this one, but using that description.